### PR TITLE
fix: regression test failed [DHIS2-5001]

### PR DESCRIPTION
**Pull request by d-bernat**

### CODE REVIEW

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java [hunk 1]
```diff
@@ -191,8 +191,8 @@ public boolean isAggregatable( AggregationType aggregationType )
         {
             return false;
         }
-
-        if ( this == TEXT || this == LONG_TEXT || this == LETTER )
+        // ?Only TEXT can have AggregationType.NONE
```
Comment (maikelarabori) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java:194]: I think that there is an additional "?"

**Review by maikelarabori — APPROVED** [review]
Looks good @d-bernat , just a small comment.

Cheers,
Maikel

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java (hunk 1 above)
Comment (d-bernat) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java:194]: :-) I just said I am not sure, but I will remove it

Comment (d-bernat) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java]: done

**Review by larshelge — APPROVED** [review]

**Pull request merged**